itests/pom.xml

```diff
@@ -117,6 +119,11 @@
           <type>pom</type>
           <scope>provided</scope>
         </dependency>
+        <dependency>
+		    <groupId>biz.aQute.bnd</groupId>
+		    <artifactId>biz.aQute.tester</artifactId>
+		    <version>${bnd.version}</version>
+		</dependency>
```

Comment:
got some weird indentation going on

Reply:
See if you can format this whole file.

Reply:
Actually, just format the parts that you are changing. Don't want to make this more difficult than necessary for other reviewers.

Reply:
OK, I fixed the formatting and rebased again against the current openhab-addons/2.5.x branch as of 3:45AM March 19th GMT.